static/styles/rewards/claim-table.css

```diff
@@ -168,8 +170,15 @@ table[data-make-claim-rendered] tr#additional-details-border + tr > * {
 table #additionalDetailsTable {
   opacity: 0;
   pointer-events: none;
-  transform: translate(-50%, -90px);
+  transform: translate(-50%, -133px);
```

Comment:
This doesn't look right.

Reply:
Humm let me look for a better way to do it then